### PR TITLE
Added necessary tooltips to icons in new frontend #2693

### LINKED ISSUE #2693: UI feedback: Add tooltips to all the new icons
**Describe the solution you'd like**
With the switch to the new UI, there are plenty of new icons for many different actions. Many actions still miss tooltips if they have no description next to it.

Icons I found that need a tooltip (incomplete list):
- [ ] Trigger search (search bar magnifying glass)
- [ ] Tooltip on the timeline chip for truncated timeline names
- [ ] Save search action
- [ ] Time chart icon
- [ ] select columns icon
- [ ] more settings icon
- [ ] star event
- [ ] tag event
- [ ] action menu
- [ ] Copy icon in the event details
- [ ] Aggregation icon in the event details
- [ ] Icons in the graph view
- [ ] Tooltip of truncated data in the Threat Intelligence table (left panel)
- [ ] Tooltip for the + icons in the left panel
- [ ] Tooltip for the manage icons in the left panel
- [ ] Icons in the Threat Intelligence feature

### PR DESCRIPTION
**Pull request by bryankhor07**
+ What existing problem does this PR solve?
It adds tooltips to icons that need a short description.

**Checks**

- [x] All tests succeed.
- [x] Unit tests added.
- [x] e2e tests added.
- [x] Documentation updated.

**Closing issues**

Closes #2693 

Co-authors: @lemonshark05 @gstgrace

### CODE REVIEW

File: timesketch/frontend-ng/src/views/Home.vue [hunk 1]
```diff
@@ -33,7 +33,7 @@ limitations under the License.
         <template v-slot:activator="{ on, attrs }">
           <v-avatar>
             <v-btn small icon v-bind="attrs" v-on="on">
-              <v-icon>mdi-dots-vertical</v-icon>
+              <v-icon title="Action Menu">mdi-dots-vertical</v-icon>
```
Comment (jkppr) [inline timesketch/frontend-ng/src/views/Home.vue:36]:
```suggestion
              <v-icon title="Sketch menu">mdi-dots-vertical</v-icon>
```

File: timesketch/frontend-ng/src/views/Sketch.vue [hunk 2]
```diff
@@ -63,7 +64,7 @@ limitations under the License.
       ]"
     >
       <v-btn icon @click="toggleLeftPanel">
-        <v-icon>mdi-menu</v-icon>
+        <v-icon title="Manage">mdi-menu</v-icon>
```
Comment (jkppr) [inline timesketch/frontend-ng/src/views/Sketch.vue:67]:
```suggestion
        <v-icon title="Toggle left panel">mdi-menu</v-icon>
```

**Review by jkppr — CHANGES_REQUESTED** [review]
First round review:
* Thanks for contributing and tackling the tooltip task.
* Please consider working from a separate branch in your fork and not from master.
* Please keep your PR clean and do not submit files that are not necessary or related to the fix you are working on.
* Please see the comments below.

**Comment by gstgrace** [comment]
Hi @jkppr , thanks for checking in! Yes, we are still revising the code based on the feedback.

**Comment by jkppr** [comment]
I forked this PR and finished it, since some UX guidance changed mid-way and we want to have it ready for the next release in December. Therefore closing this PR in favor of #2983 .

Thanks for contributing @gstgrace @bryankhor07  @lemonshark05

**Pull request closed without merging**